devtools-ci-index.yaml

```diff
@@ -555,7 +555,10 @@
             branches:
                 - openshift-connector
     triggers:
-        - github
+        - github-pull-request:
+            white-list-target-branches: 
+              - openshift-connector
+            permit-all: true
```

Comment:
kept github as well, now there are 2 triggers - github (triggered on push to repo), github PR (triggered on PR to repo when a 'test PR' comment is written).